### PR TITLE
Add ascii/latin1/unicode character generators

### PR DESCRIPTION
**Pull request by moodmosaic**
In line with https://github.com/hedgehogqa/haskell-hedgehog/pull/73/.

### CODE REVIEW

File: src/Hedgehog/Gen.fs [hunk 1]
```diff
@@ -354,7 +368,7 @@ module Gen =
 
     /// Generates a random string.
     let string : Gen<string> =
-        choice [alphaNum; charBounded] |> string'
+        choice [alphaNum; unicodeAll] |> string'
```
Comment (jystic) [inline src/Hedgehog/Gen.fs]: Maybe this should be just `unicode` given the surrogates won't be a legal string unless they're combined correctly? (sorry I don't know that much about unicode surrogates, but I get the impression this wouldn't be a valid string as such)

Comment (moodmosaic) [inline src/Hedgehog/Gen.fs]: Hmm... True, `unicode` is better default, I'll change it.

And, I think, we need something similar in the Haskell version? Right now it uses min/max bounds of char, no?

Comment (jystic) [inline src/Hedgehog/Gen.fs]: I don't think there is a haskell version, the haskell `Gen.string` takes a `Gen<char>` as an argument

Comment (moodmosaic) [inline src/Hedgehog/Gen.fs]: Done now in 4979fb5.

Comment (jystic) [inline src/Hedgehog/Gen.fs]: hmm, I wonder if we should just rename `string'` -> `string` and delete this, let people make their own choices for characters?

Comment (moodmosaic) [inline src/Hedgehog/Gen.fs]: *That* was exactly what I was thinking right now... We're going to have `Gen.fs` in line with the Haskell one anyway, so good idea. I'll undo 4979fb5 also.

Comment (moodmosaic) [inline src/Hedgehog/Gen.fs]: OK, I just pushed f39c328.

**Comment by jystic** [comment]
👍 sorry for the delay

**Pull request merged**